crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -459,6 +459,45 @@ impl<'s> Gizmos<'s> {
         }
     }
 
+    /// Draw an arc, which is a part of the circumference of a circle.
+    ///
+    /// # Example
+    /// ```
+    /// # use bevy_gizmos::prelude::*;
+    /// # use bevy_render::prelude::*;
+    /// # use bevy_math::prelude::*;
+    /// # use std::f32::consts::PI;
+    /// fn system(mut gizmos: Gizmos) {
+    ///     gizmos.arc_2d(Vec2::ZERO, 0., PI / 4., 1., Color::GREEN);
+    ///
+    ///     // Arcs have 32 line-segments by default.
+    ///     // You may want to increase this for larger arcs.
+    ///     gizmos
+    ///         .arc_2d(Vec2::ZERO, 0., PI / 4., 5., Color::RED)
+    ///         .segments(64);
+    /// }
+    /// # bevy_ecs::system::assert_is_system(system);
+    /// ```
+    #[inline]
+    pub fn arc_2d(
+        &mut self,
+        position: Vec2,
+        direction_angle: f32,
+        arc_angle: f32,
```

Comment:
Should we call this `arc_length` instead? I'm not much of a math person, but that seems like a more common term for this.

Reply:
I think `arc_length` might be confusing. The argument needed here is definately angle. The user will usually use a fraction of `PI` like `PI / 4.` or `PI / 2.`.

Reply:
Oops, you're totally right.